.doc_gen/metadata/bedrock_metadata.yaml

```diff
@@ -0,0 +1,18 @@
+# zexi 0.4.0
+bedrock-ListFoundationModels:
```

Comment:
This should be `bedrock_ListFoundationModels` (underscore, not -)

Reply:
Done